app/scripts/modules/app/vite.config.js

```diff
@@ -0,0 +1,46 @@
+import dotenv from 'dotenv';
+import fs from 'fs';
+import path from 'path';
+import strip from 'rollup-plugin-strip-code';
+import { defineConfig } from 'vite';
+
+import angularTemplateLoader from '@spinnaker/scripts/helpers/rollup-plugin-angularjs-template-loader';
+
+const DECK_ROOT = path.resolve(`${__dirname}/../../../..`);
+const NODE_MODULE_PATH = path.resolve(`${DECK_ROOT}/node_modules`);
+
+const envLocalFilePath = path.resolve(`${__dirname}/.env.local`);
+if (fs.existsSync(envLocalFilePath)) {
+  dotenv.config({
+    path: envLocalFilePath,
+  });
+}
+
+export default defineConfig({
+  clearScreen: false,
```

Comment:
Do we want to keep this in dev? (I'm not sure)

Reply:
I definitely want to keep it in dev 'cos it clears some of the errors which is not ideal.